contentcuration/contentcuration/frontend/shared/views/LicenseDropdown.vue

```diff
@@ -23,19 +23,38 @@
           :class="{ 'with-trailing-input-icon': box }"
```

Comment:
We had this class to include the trailing icon for the about licenses modal. As we are removing this, we need to remove also this line and/or fix these alignment styles, and all the related styles here https://github.com/LianaHarris360/studio/blob/0cefdcec13a859cb739914fdb6cd0fabb65c99d9/contentcuration/contentcuration/frontend/shared/views/LicenseDropdown.vue#L242, as this cause this little misalign in the dropdown icon.

![image](https://github.com/user-attachments/assets/50d3aa03-d842-48f4-b497-d8e7cc2576a6)

Reply:
Thanks for pointing this out, I was wondering about this misalignment but didn't initially see where it was being set. Now I can get it fixed!

Reply:
The input selection arrow alignment is now fixed :)
![EditSourceModal](https://github.com/user-attachments/assets/aa27fc59-e0db-4494-bb0d-9b2870875979)

Reply:
Hi Liana! It seems we now have this little bug when we select an option with less width :sweat_smile: :

![image](https://github.com/user-attachments/assets/18eaa8c8-2228-4aed-a893-c8f553572cb3)

![image](https://github.com/user-attachments/assets/050189f6-9eac-44da-a15b-ca3d9d03cb0a)

Reply:
I believe this bug was present before these changes were made. I attempted to edit the source on unstable and encountered the same issue. Adding a minimum width fixes it! I also noticed it had a slightly darker background than the rest of the inputs in the EditSource modal and updated that as well.

https://github.com/user-attachments/assets/11ab4fd7-e798-49ca-80a8-5a5772593086